view/works.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Works</title>
+    <link rel="stylesheet" href="/public/css/works.css">
```

Comment:
Same thing for linking the `css` file, you don't need the `/public` otherwise it will not found the file:

```
<link rel="stylesheet" href="/css/photo.css">
```